crates/ink/codegen/src/generator/trait_def/call_builder.rs

```diff
@@ -347,6 +347,7 @@ impl CallBuilder<'_> {
         let mut_tok = message.mutates().then(|| quote! { mut });
         quote_spanned!(span =>
             #[allow(clippy::type_complexity)]
+            #( #attrs )*
```

Comment:
Maybe you want to use `cfg_attrs` here because it is generated part of the trait(not expected by the user)?